towhee/runtime/runtime_pipeline.py

```diff
@@ -36,11 +40,13 @@ def __init__(self,
                  nodes: Dict[str, NodeRepr],
                  edges: Dict[str, Any],
                  operator_pool: 'OperatorPool',
-                 thread_pool: 'ThreadPoolExecutor'):
+                 thread_pool: 'ThreadPoolExecutor',
+                 time_profiler: 'TimeProfiler' = TimeProfiler()):
```

Comment:
Use None as the default value, and in `__init__`,  create TimeProfiler if the input is None.